CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Release 0.3.0
+Allow fetching all facts.
+
+### Added
+- Fetch all facts if no fact to fetch is provided.
```

Comment:
It also changed the output format. Instead of just displaying the value, it returns a JSON object mapping facts to values (even for a single fact).